insights/parsers/db2.py

```diff
@@ -0,0 +1,53 @@
+"""
+DB2 commands
+============
+
+Shared parsers for parsing output of the commands of IBM DB2
+
+Db2ls - command ``db2ls -a -c``
+-------------------------------
+"""
+from insights import parser, CommandParser
+from insights.parsers import SkipException
+from insights.specs import Specs
+
+
+@parser(Specs.db2ls_a_c)
+class Db2ls(CommandParser, list):
+    """
+    Parse the output of `db2ls -a -c` into a list of dictionaries.
+
+    Typical output of this command::
+
+        #PATH:VRMF:FIXPACK:SPECIAL:INSTALLTIME:INSTALLERUID
+        /opt/ibm/db2/V11.5:11.5.6.0:0 ::Fri Jan 14 20:20:07 2022 CST :0
+        /opt/ibm/db2/V11.5_01:11.5.7.0:0 ::Fri Feb 11 10:34:51 2022 CST :0
+
+    Examples:
```

Comment:
Please leave an empty line so that the example output is formatted,


Before
---------
![db2](https://user-images.githubusercontent.com/909861/153810417-965d7903-1640-48b6-a1d8-7e712f3da2a5.png)

After
-------
![db2-1](https://user-images.githubusercontent.com/909861/153810575-87de3bb5-54d2-41f2-9ece-5c5d9f04c1da.png)

Reply:
Thanks for reviewing, I fixed it in the new commit by adding an indented block.